self-compile

```diff
@@ -11,6 +11,7 @@ SV_BUILD_TARGET=${SV_BUILD_TARGET:-"silver:compiler:composed:Default"}
 
 export ANT_OPTS=-Xss10M
 
+cd "$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
```

Comment:
What does this do?

Reply:
Oh, just realized the `realpath` is superfluous; but it's so I can run `../../self-compile` from e.g. `test/silver_features`